src/sig_stfl/sig_stfl.h

```diff
@@ -54,6 +54,74 @@ OQS_API int OQS_SIG_STFL_alg_count(void);
  */
 OQS_API int OQS_SIG_STFL_alg_is_enabled(const char *method_name);
 
+/**
+ * @brief OQS_SECRET_KEY object for stateful signature schemes
+ */
+typedef struct OQS_SECRET_KEY OQS_SECRET_KEY;
+
+typedef struct OQS_SECRET_KEY {
+    /** A local ordinal representing the LMS parameter of the signature scheme. */
+    uint32_t oid;
+
+    /** Printable string representing the name of the signature scheme. */
+    const char *method_name;
```

Comment:
Let's replace this with a point to the OQS_SIG_STFL object, which contains the oid and method_name.

Reply:
Done.

Reply:
When do you think this pointer would get set?

Reply:
Good question. I guess it would be helpful to see some test code doing these things in sequence, and then I guess it would be clear when to set it...